Cargo.toml

```diff
@@ -43,19 +43,17 @@ js-component-bindgen = { path = "./crates/js-component-bindgen" }
 serde = "1.0.203"
 serde_json = "1.0.118"
 structopt = "0.3.26"
-wasm-encoder = "0.219.1"
-wasm-metadata = "0.219.1"
-wasmparser = "0.219.1"
-wasmprinter = "0.219.1"
-wasmtime-environ = { version = "27.0.0", features = [
-    "component-model",
-    "compile",
-] }
+wasm-encoder = "0.223.0"
+wasm-metadata = "0.223.0"
+wasmparser = "0.223.0"
+wasmprinter = "0.223.0"
+wit-parser = "0.223.0"
```

Comment:
nit: I suggest preserving the original order, to make it easier to merge future changes.